pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -72,7 +74,10 @@ func nodeSelectorMatchesThisNode(cl client.Client, eventObject runtime.Object) b
 func forThisNodePredicate(cl client.Client) predicate.Funcs {
 	return predicate.Funcs{
 		CreateFunc: func(createEvent event.CreateEvent) bool {
-			return nodeSelectorMatchesThisNode(cl, createEvent.Object)
+			if nodeSelectorMatchesThisNode(cl, createEvent.Object) {
+				return true
+			}
+			return false
```

Comment:
Why did you change this? Isn't it exactly what the original did?

Reply:
Leftover from some changes, will go back to old version.